test/CucumberLanguageServer.test.ts

```diff
@@ -50,7 +50,7 @@ describe('CucumberLanguageServer', () => {
 
     const initializeParams: InitializeParams = {
       rootUri: `file://${process.cwd()}`,
-      processId: 1,
+      processId: 'N/A' as unknown as number, // This id is used by vscode-languageserver. Forcing this to be a string so that the watchdog responsible for watching this process does not run.
```

Comment:
Instead of using a string, would `NaN` work as well?

Reply:
Oh, indeed it does -- good idea. Updated here 658f17727e32807f2a82dce89eed6bfb614315d0